spec.html

```diff
@@ -8389,17 +8821,34 @@ <h1>MakeMethod ( _F_, _homeObject_ )</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-define-ordinary-method" aoid="DefineOrdinaryMethod">
+      <h1>DefineOrdinaryMethod ( _key_, _homeObject_, _closure_, _enumerable_ )</h1>
+      <emu-alg>
+        1. Perform SetFunctionName(_closure_, _key_).
+        1. If _key_ is a Private Name, then
+          1. Assert: _key_ does not have a [[Kind]] field.
+          1. Set _key_.[[Kind]] to `"method"`.
+          1. Set _key_.[[Value]] to _closure_.
+          1. Set _key_.[[Brand]] to _homeObject_.
+        1. Else,
+          1. Let _desc_ be the PropertyDescriptor { [[Value]]: _closure_, [[Writable]]: *true*, [[Enumerable]]: _enumerable_, [[Configurable]]: *true* }.
+          1. Perform ? DefinePropertyOrThrow(_homeObject_, _key_, _desc_).
+      </emu-alg>
+    </emu-clause>
+
     <emu-clause id="sec-setfunctionname" aoid="SetFunctionName">
       <h1>SetFunctionName ( _F_, _name_ [ , _prefix_ ] )</h1>
-      <p>The abstract operation SetFunctionName takes arguments _F_ (a function object) and _name_ (a property key) and optional argument _prefix_ (a String). It adds a *"name"* property to _F_. It performs the following steps when called:</p>
+      <p>The abstract operation SetFunctionName takes arguments _F_ (a function object) and _name_ (a property key or Private Name) and optional argument _prefix_ (a String). It adds a *"name"* property to _F_. It performs the following steps when called:</p>
       <emu-alg>
         1. Assert: _F_ is an extensible object that does not have a *"name"* own property.
-        1. Assert: Type(_name_) is either Symbol or String.
+        1. Assert: Type(_name_) is either Symbol, String, or Private Name.
         1. Assert: If _prefix_ is present, then Type(_prefix_) is String.
         1. If Type(_name_) is Symbol, then
           1. Let _description_ be _name_'s [[Description]] value.
           1. If _description_ is *undefined*, set _name_ to the empty String.
           1. Else, set _name_ to the string-concatenation of *"["*, _description_, and *"]"*.
+        1. Else if _name_ is a Private Name, then
```

Comment:
For consistency, I think

```suggestion
        1. Else if Type(_name_) is Private Name, then
```